sim/firesim-lib/src/main/cc/bridges/tracerv.h

```diff
@@ -5,8 +5,12 @@
 #include "bridges/bridge_driver.h"
 #include <vector>
 
+// TODO: get this automatically
```

Comment:
TRACERV will just be a one-per-core bridge after multiclock. I'll remove this then